jupyterlab_wipp/jupyterlab_wipp/wipp.py

```diff
@@ -99,7 +100,66 @@ def get_image_collections_all_pages(self):
 
     def get_image_collections(self):
         return [collection.json for collection in sum(self.get_image_collections_all_pages(), [])]
+
+    def search_image_collections_summary(self, name):
```

Comment:
Can you add comments before each function with a brief summary and explaining the expected inputs and outputs?

Reply:
Done